docs/v0.9/zh/getting-started/installation/greptimedb-cluster.md

```diff
@@ -2,68 +2,153 @@
 
 GreptimeDB 可以运行于 [cluster](/contributor-guide/overview.md) 模式以支持水平扩展。
 
-## 安装 gtctl 并运行 playground
+## 使用 Docker Compose
 
-[gtctl](https://github.com/GreptimeTeam/gtctl) 是用于管理 GreptimeDB 的命令行工具。 你可以用如下命令进行安装（只适用于 Linux 和 macOS）:
+### 前置条件
+
+使用 Docker Compose 是运行 GreptimeDB 集群的最简单方法。开始之前，请确保已经安装了 Docker。
+
+### 步骤 1: 下载 Docker Compose 的 YAML 文件
 
 ```
-curl -fsSL https://downloads.greptime.cn/releases/scripts/gtctl/install.sh | sh -s -- -s aws
+wget https://raw.githubusercontent.com/GreptimeTeam/greptimedb/<%greptimedb-version%>/docker/docker-compose/cluster-with-etcd.yaml
```

Comment:
suggestion: Save the `cluster-with-etcd.yaml` file in CN region s3 and change command `wget https://downloads.greptime.cn/xxx/cluster-with-etcd.yaml`.